ext/AvizExt/theme.jl

```diff
@@ -82,4 +85,118 @@ function scenario_colors!(obs_color::Observable, color_map::Vector, scen_types::
+    clusters::Vector{Int64}, unique_colors::Vector{RGBA{Float32}}
+)::Vector{RGBA{Float32}}
+    unique_clusters::Vector{Int64} = sort(unique(clusters))
+    colors_map::Dict{Int64,RGBA{Float32}} = Dict(
+        c => unique_colors[i] for (i, c) in enumerate(unique_clusters)
+    )
+
+    colors::Vector{RGBA{Float32}} = Vector{RGBA{Float32}}(undef, length(clusters))
+    for (idx_c, cluster) in enumerate(clusters)
+        colors[idx_c] = colors_map[cluster]
+    end
+
+    return colors
+end
+function cluster_colors(clusters::Vector{Int64})::Vector{RGBA{Float32}}
+    unique_colors::Vector{RGBA{Float32}} = categorical_colors(
+        :seaborn_bright, length(unique(clusters))
+    )
+    return cluster_colors(Int64.(clusters), unique_colors)
+end
+function cluster_colors(clusters::BitVector)::Vector{RGBA{Float32}}
+    if unique(clusters) == [0]
+        return fill(parse(Colorant, :red), length(clusters))
+    elseif unique(clusters) == [1]
+        return fill(parse.(Colorant, :blue), length(clusters))
+    end
+    unique_colors::Vector{RGBA{Float32}} = parse.(Colorant, [:red, :blue])
+    return cluster_colors(Int64.(clusters), unique_colors)
+end
+
+"""
+    cluster_alphas(clusters::Vector{Int64})::Vector{Float64}
+    cluster_alphas(clusters::BitVector)::Vector{Float64}
+
+Get color alphas for each cluster weighted by number of scenarios.
+
+# Arguments
+- `clusters` : Vector with scenario cluster numbers
+
+# Returns
+Vector with one color alpha for each cluster.
+"""
+function cluster_alphas(clusters::Vector{Int64})::Dict{Int64,Float64}
+    alphas::Dict{Int64,Float64} = Dict()
+
+    for (i, cluster) in enumerate((unique(clusters)))
+        n_scens::Int64 = count(clusters .== cluster)
+        base_alpha::Float64 = 1.0 / (n_scens * 0.05)
+        alphas[i] = max(min(base_alpha, 0.6), 0.1)
+    end
+
+    return alphas
+end
+function cluster_alphas(clusters::BitVector)::Dict{Int64,RGBA{Float32}}
+    return cluster_alphas(Int64.(clusters))
+end
+
+"""
+    cluster_labels(clusters::Vector{Int64})::Vector{String}
+    cluster_labels(clusters::BitVector)::Vector{String}
+    cluster_labels(clusters::Vector{Int64}, data::AbstractVector{<:Real})::Vector{String}
+    cluster_labels(clusters::BitVector, data::AbstractVector{<:Real})::Vector{String}
+    cluster_labels(cluster_names::Vector{String}, data::AbstractVector{<:Real})::Vector{String}
+
+Get labels for each cluster.
+
+# Arguments
+- `clusters` : Vector with scenario cluster numbers
+
+# Returns
+Vector of labels for each cluster.
+"""
+function cluster_labels(clusters::Vector{Int64})::Vector{String}
+    return "Cluster " .* string.(unique(clusters))
+end
+function cluster_labels(clusters::BitVector)::Vector{String}
+    return [BINARY_LABELS[cluster] for cluster in unique(clusters)]
+end
+function cluster_labels(
+    clusters::Vector{Int64}, data::AbstractVector{<:Real}
+)::Vector{String}
+    cluster_names::Vector{String} = ["Cluster $(cluster)" for cluster in clusters]
```

Comment:
```suggestion
    cluster_names::Vector{String} = cluster_labels(clusters)
```

or if non-unique elements are allowed:

```julia
cluster_names = "Cluster" .* string.(clusters)
```